docs/content/en/docs/reference/clusterspec/cloudstack.md

```diff
@@ -236,7 +236,7 @@ Number of etcd members
 Refers to the Kubernetes object with CloudStack specific configuration for your etcd members. See `CloudStackMachineConfig Fields` below.
 
 ### kubernetesVersion (required)
-The Kubernetes version you want to use for your cluster. Supported values: `1.24`, `1.23`, `1.22`, `1.21`
+The Kubernetes version you want to use for your cluster. Supported values: `1.24`, `1.23`, `1.22`
```

Comment:
Missing 1.25 and 1.26?

Reply:
We don't support 1.25 and 1.26 on CloudStack yet